components/bl-totop-component/src/index.js

```diff
@@ -0,0 +1,66 @@
+import { useRef, useMemo, useEffect } from "react";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const AxisClassesMap = {
+  X: { left: "start-0", right: "end-0", center: "start-50 translate-middle-x" },
+  Y: { top: "top-0", bottom: "bottom-0", center: "top-50  translate-middle-y" },
+};
+
+export default function TotopComponent({ component }) {
+  const { classList, display, offset, position, backgroundColor, color, size, iconSize, indentX, indentY, element } = component;
```

Comment:
offset, position, backgroundColor, color, size, iconSize, indentX, indentY - we must add logic handlers for that(where it possible)